modules/heat_conduction/doc/content/source/userobjects/GrayLambertSurfaceRadiation.md

```diff
@@ -0,0 +1,174 @@
+\end{equation}
+
+At this point it is convenient to represent the radiosity, temperature, irradiation,
+and emissivity on each distinct surface $i$ by a suitable average of the actual distribution.
+We leave out the $\vec{r}$ argument and add an $i$ index by which we indicate that this
+quantity has been averaged over the extent of surface $i$.
+
+A second balance equation in addition to [eq:radiosity_gray] is obtained by relating
+the irradiation onto surface $i$ to the radiosities of all other surfaces:
+
+\begin{equation}\label{eq:second_balance}
+  H_i = \sum\limits_{j=1}^n F_{i,j} J_j,
+\end{equation}
+where $A_i$ is the area of surface $i$, $n$ is the total number of surfaces, and $F_{i,j}$ is the view
+factor from surface $i$ to surface $j$. Eliminating $H_i$ from [eq:second_balance] using
+[eq:radiosity_gray] gives:
+
+\begin{equation}\label{eq:fixed_q}
+  \sum\limits_{j=1}^n \left( \delta_{i,j} - F_{i,j }\right) J_j = \dot{q}_i.
+\end{equation}
+
+This result is used for computing $J_j$ on surfaces where $\dot{q}_i$ is known. This is the case for
+`ADIABATIC` surfaces, where $\dot{q}_i = 0$.
+
+A more convenient relationship is derived for surfaces where $T_i$ is either known (`FIXED_TEMPERATURE`) or
+computed (`VARIABLE_TEMPERATURE`). From [eq:radiosity_gray] and [eq:heat_balance], we obtain:
+
+\begin{equation}
+ \dot{q}_i = \frac{\epsilon_i}{1 - \epsilon_i} \left( \sigma T_i^4 - J_i \right).
+\end{equation}
+Then we use this equation in [eq:fixed_q] to eliminate $\dot{q}_i$:
+\begin{equation}\label{eq:fixed_T}
+ \sum\limits_{j=1}^n \left( \delta_{i,j} - (1 - \epsilon_i) F_{i,j }\right) J_j = \epsilon_i \sigma T_i^4.
+\end{equation}
+
+## Relationship of Net Radiation Method with Radiative Transport
+
+The radiative transport equation is formulated in terms of the spectral intensity $I(\vec{r}, t, \lambda, \hat{\Omega})$ with $\lambda$ being the photon's wavelength and $\hat{\Omega}$ being the direction
+of propagation. The physical meaning of the spectral intensity is that the energy transported through
+area $dA$ at $\vec{r}$, during time interval $dt$ at time $t$, along direction within a cone $d\hat{\Omega}$ about $\hat{\Omega}$ and with wavelengths within $d \lambda$ about $\lambda$ is:
+
+\begin{equation}
+  \text{Energy} = I(\vec{r}, t, \lambda, \hat{\Omega}) dt dA |\vec{n} \cdot \Omega| d\Omega d\lambda,
+\end{equation}
+
+where $\vec{n}$ is the normal on face $dA$.
+The intensity is computed by a code that solves the radiative transfer equation, e.g. Rattlesnake within the MOOSE framework. The solution of the radiative transfer equation is coupled to heat conduction within solid domains
+through the balance at the surface [eq:heat_balance], but now we evaluate the net heat flux from the spectral intensity:
+
+\begin{equation}
+  \dot{q}(\vec{r}) = \int\limits_{4 \pi} d\hat{\Omega} \int\limits_{0}^{\infty} d\lambda (\vec{n} \cdot \hat{\Omega}) I(\vec{r}, t, \lambda, \hat{\Omega}).
+\end{equation}
```

Comment:
This is the boundary condition for thermal conduction. Can we also give the boundary condition for radiative transfer here with emissivity and reflectivity for completeness?

Reply:
Yes, I'll add that here.